### DISCUSSION TITLE
do i need alova if i use react query?

### DISCUSSION THREAD

**Discussion by cStralpt** [alovajs/alova#194] [General]
i use react query for my server state management, it supporting caching too,if i use alova does it still worth it to use all of its feature? or...
just use fetch api instead?

**Comment by JOU-amjs** [comment]
I think no, many features of alova are duplicated with react-query such as cache, states management...
but, the goal of alova is different from react-query, and alova is a request strategy library and also provides many features that react-query has no.
so you need to decide for yourself which one is more suitable for your project.
